joomla/joomla-cms
# [4.0] Hide column default for trashed stages

### Summary of Changes
Hide the column "Default" in stages list, when Filter is "trashed", so that trashed items cannot be set as default
![trashednew](https://user-images.githubusercontent.com/1035262/46587298-81655700-ca8a-11e8-8761-a2455ad0b9cf.PNG)



### Testing Instructions
Go to the stages overview of a workflow. 
Trash one or more items.
Use the filter trashed and see the overview of trashed items.

### Expected result
The column default is now invisible if the filter is "trashed"

### Actual result
The column "Default" is visible and you can set a trashed item to default.
![trashed2default](https://user-images.githubusercontent.com/1035262/46587241-7e1d9b80-ca89-11e8-8b57-db8d9e9a7c28.PNG)

### Documentation Changes Required
no
